packs/octopusdeploy/actions/get_deployments.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: get_deployments
```

Comment:
Minor style / consistency thing - we tend prefix actions which return a list with `list_` (e.g. `list_deployments`) and methods which return a single item with `get_` (e.g. `get_deployment`).